src/components/Checkbox/Checkbox.tsx

```diff
@@ -40,89 +40,96 @@ export default function Checkbox(props: ICheckboxProps) {
   } = props
 
   return (
-    <React.Fragment>
-      <Label
-        htmlFor={name}
+    <div
+      sx={{
+        display: "inline-flex",
+        position: "relative",
+        alignItems: "center",
+        opacity: disabled ? 0.4 : 1,
+        cursor: disabled ? "auto" : "pointer",
+      }}
+    >
+      <input
+        {...restProps}
+        type="checkbox"
+        id={name}
+        name={name}
+        title={name}
```

Comment:
I think it would be better to no set `title` as `name`, as the `name` prop is being used for the input name and id, which usually don't have a value that is supposed to be seen by users (as a native tooltip appears when you leave your mouse over an element with a title attribute). I can see myself passing `name="isActive"` for example, which is not a text I would want my user to see.

If the dev needs to set the title so there's a tooltip on mouse-over he could still use the `{...restProps}`.